src/main/java/me/gnat008/perworldinventory/PerWorldInventory.java

```diff
@@ -38,6 +38,7 @@
 import java.util.logging.Logger;
 
 public class PerWorldInventory extends JavaPlugin {
+	public static final int CONFIG_VERSION = 4;
```

Comment:
Could you change this to use spaces and not tab? Other than that, this looks good!